src/sourceMaps/sourceMap.ts

```diff
@@ -63,8 +64,11 @@ export class SourceMap {
 
         // Rewrite sm.sources to same as this._sources but forward slashes and file url
         sm.sources = this._sources.map(sourceAbsPath => {
-            // Convert to file: url. After this, it's a file URL for an absolute path to a file on disk with forward slashes.
-            return utils.pathToFileURL(sourceAbsPath);
+            // Convert to file:/// url. After this, it's a file URL for an absolute path to a file on disk with forward slashes.
+            // We lowercase so authored <-> generated mapping is not case sensitive.
+            const lowerCaseSourceAbsPath = sourceAbsPath.toLowerCase();
```

Comment:
Weird that I can't make a comment on a line you didn't change. But since `generatedPositionFor()` requires a lowercase path now, that should either have a comment, or `generatedPositionFor` should make `source` lowercase.